src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -1349,21 +1512,158 @@ mod tests {
             fn is_yellow_pressurised(&self) -> bool {
                 self.hydraulics.is_yellow_pressurised()
             }
+
+            fn ac_bus_1_power_state(&mut self, bus_is_alive: bool) {
+                self.is_ac_1_alive = bus_is_alive;
+            }
+            fn ac_bus_2_power_state(&mut self, bus_is_alive: bool) {
+                self.is_ac_2_alive = bus_is_alive;
+            }
+
+            fn dc_ground_service_power_state(&mut self, bus_is_alive: bool) {
+                self.is_dc_ground_service_alive = bus_is_alive;
+            }
+
+            fn ac_ground_service_power_state(&mut self, bus_is_alive: bool) {
+                self.is_ac_ground_service_alive = bus_is_alive;
+            }
+
+            fn dc_bus_2_power_state(&mut self, bus_is_alive: bool) {
+                self.is_dc_2_alive = bus_is_alive;
+            }
+            fn dc_ess_power_state(&mut self, bus_is_alive: bool) {
+                self.is_dc_ess_alive = bus_is_alive;
+            }
```

Comment:
**issue**: Rust setters should start with `set_`.
**suggestion**: Write:
```rust
fn set_dc_bus_2_is_powered(&mut self, is_powered: bool) {
    self.dc_bus_2_is_powered = is_powered;
}
```
**issue**: Inconsistent (missing) whitespace between functions.